gpkitmodels/GP/aircraft/motor/motor.py

```diff
@@ -0,0 +1,103 @@
+"Electric motor model "
+from gpkit import Model, parse_variables
+from gpkit.constraints.tight import Tight as TCS
+from gpkitmodels.GP.aircraft.prop.propeller import Propeller, ActuatorProp
+from gpkitmodels import g
+
+class MotorPerf(Model):
+    """ Electric Motor Performance Model
+
+    Variables
+    ---------
+    Pshaft                  [kW]            motor output shaft power
+    Pelec                   [kW]            motor input shaft power
+    etam                    [-]             motor efficiency
+    Q                       [N*m]           torque
+    omega                   [rpm]           propeller rotation rate
+    i                       [amps]          current
+    v                       [V]             woltage
+    i0         4.5          [amps]          zero-load current
+    Kv         29           [rpm/V]         motor voltage constant
+    R          .033         [ohms]          internal resistance
+    """
+    def setup(self, static, state):
+        exec parse_variables(MotorPerf.__doc__)
+
+        return [Pshaft == Q*omega,
+                Pelec == v*i,
+                etam == Pshaft/Pelec,
+                static.Qmax >= Q,
+                v <= static.V_max,
+                TCS([i >= Q*Kv+i0]),
+                TCS([v >= omega/Kv + i*R])
```

Comment:
Are you worried about these not being tight?

Reply:
I was at one point.  It depends on how the model is being used.  Is there a high cost to having that check?

Reply:
No, but in general I think we should only have TCS when there's a possible use case where it shouldn't be tight.  Let's leave it for now.